conf/scylla.yaml

```diff
@@ -574,6 +574,23 @@ force_schema_commit_log: true
 # documentation, section 'Enabling Raft in ScyllaDB 5.2 and further', for the procedure.
 consistent_cluster_management: true
 
+# In materialized views, restrictions are allowed only on the view's primary key columns.
```

Comment:
I think you copied a comment from a completely unrelated test??

Reply:
Or, wait, what is this commit?
Maybe you should rebase...

Reply:
This was auto-rebase commit made by github, which actually turned out to be a merge commit.